examples/angular/src/app/lib/components/no-component/no-component.component.ts

```diff
@@ -0,0 +1,20 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { DotCMSContentlet } from '../../models';
+
+/**
+ * This is part of the React SDK. We should let user have their own callback component
```

Comment:
React?

Reply:
I copied and pasted the component. I'll remove the comment.